### PR TITLE
script to generate popularities.json

### PR DESCRIPTION
**Pull request by peterbe**
This makes it possible to update the `files/popularties.json` in the `mdn/content` repo. It's a script and also my notes about how to generate it.

### CODE REVIEW

**Comment by peterbe** [comment]
@escattone @fiji-flo Perhaps we can split up. If Florian can review the code itself and Ryan can sanity test that he can use it I think it should be good to go.

**Comment by peterbe** [comment]
I used this code to generate https://github.com/mdn/content/pull/1773

File: docs/popularities.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Popularities
+
+A popular page is one that has lot of pageviews. We get this from Google Analytics.
+Being popular helps search because when a certain search term matches many
+documents, too many to display all, we need to sort them to try to predict
+which one the user most probably wanted to find.
+
+To accomplish this we check in a file in the content repo called `popularities.json`
```
Comment (escattone) [inline docs/popularities.md:8]:
```suggestion
To accomplish this we check-in a file in the content repo called `popularities.json`
```

**Review by escattone — APPROVED** [review]
Thanks @peterbe, I followed your instructions in the `docs/popularities.md` file and they worked like a charm. I had just a tiny nit in the `docs/popularities.md` file.

**Pull request merged**